### PR TITLE
DOC-1912: Add v22.1.0-beta.5 release note

### PR DESCRIPTION
**Pull request by MichaelTrestman**
Fixes DOC-1912

### CODE REVIEW

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `Crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` builtins now check if user has the admin role. [#80384][#80384]
+- [#80248][#80248] [7d55af0e6][7d55af0e6]  (Raphael 'kena' Poss <knz@cockroachlabs.com>)
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:10]: I'm not sure what's happened here? Should there be text? Can you double check if this should be a note or not?

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `Crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` builtins now check if user has the admin role. [#80384][#80384]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:9]: You can remove the cap from `Crdb_internal....` 

I would link admin role here: security-reference/authorization.html#admin-role

builtins could be "built-in functions" with a link functions-and-operators.html#system-info-functions

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- [#80248][#80248] [7d55af0e6][7d55af0e6]  (Raphael 'kena' Poss <knz@cockroachlabs.com>)
+
+<h3 id="v22-1-0-beta-5-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Backups run by secondary tenants now write protected timestamp records to protect their target schema objects from garbage collection during backup execution. [#80670][#80670]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:14]: "Backups" link to take-full-and-incremental-backups.html 

Also not 100% that this note needs to be included in the release notes. If you had enough time, I would double check with engineer.

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+<h3 id="v22-1-0-beta-5-sql-language-changes">SQL language changes</h3>
+
+- The cluster settings `cloudstorage.<provider>.read.node_rate_limit` and `cloudstorage.<provider>.read.node_burst_limit` can now be used to limit throughput when reading from cloud storage during a `RESTORE` or `IMPORT`. [#80281][#80281]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:18]: link:

cluster settings --> cluster-settings.html
restore --> restore.html 
import --> import.html

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+<h3 id="v22-1-0-beta-5-bug-fixes">Bug fixes</h3>
+
+- Addressed issue where automatic encryption-at-rest data key rotation would get disabled after a node restart without a store key rotation. [#80564][#80564]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:22]: "Addressed issue.." could just adapt to "Fixed a bug..."... Maybe "would get disabled" ---> "would disable"

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+- Fixed a bug whereby the cluster version could regress due to a race condition. [#80712][#80712]
+
+<h3 id="v22-1-0-beta-5-performance-improvements">Performance improvements</h3>
+
+- Bulk ingestion of unsorted data during `IMPORT` and schema changes now uses a higher level of parallelism to send produced data to the storage layer. [#80487][#80487]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:28]: With more time, I might double check whether this needs including in release notes. 

import --> import.html
storage layer --> architecture/storage-layer.html

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+<h3 id="v22-1-0-beta-5-miscellaneous">Miscellaneous</h3>
+
+<h4 id="v22-1-0-beta-5-docker">Docker</h4>
+
+- Refactored the initialization process of the docker image to accomodate the use case with memory storage. [#80558][#80558]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:34]: I'm not sure what this "the use case with memory storage" means here? Maybe rework this note to align with what the change is and why user's need to know.

"docker" should be capped

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+- [#80641][#80641] [c02b3f015][c02b3f015] release-22.1: update cluster-ui version to v22.1.0 (Marylia Gutierrez <marylia@cockroachlabs.com>)
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:35]: Not sure what this note is? Is there a release note within the PR here?

**Review by kathancox — COMMENTED** [review]
Hey, there are few things that need to be updated before this is released. Added some links here so you can add easily

**Review by knz — COMMENTED** [review]
> Reviewed 1 of 3 files at r1, 1 of 2 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13781)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox)

___
*[`_includes/releases/v22.1/v22.1.0-beta.5.md` line 10 at r1](https://reviewable.io/reviews/cockroachdb/docs/13781#-N19MBPlGYlZKlY7TnLy-r1-10:-N1Erf9B0fvMvMUu527r:b-a6mjoc) ([raw file](https://github.com/cockroachdb/docs/blob/1456202b7e989d23b2efff5c25af03a423a62067/_includes/releases/v22.1/v22.1.0-beta.5.md#L10)):*
<details><summary><i>Previously, MichaelTrestman (Michael Trestman) wrote…</i></summary><blockquote>

There was no text provided so I thought the link was better than nothing if people wanted to look. I believe this is internal only as it affects the upgrade path used by CC. Deleted it. @knz Is there a note that should be provided, or is this best deleted from the release notes?
</blockquote></details>

correct no release note needed. You were tagged to ensure that the other PR about scram properly shows that scram is not enabled by default in 22.1 (previous release notes mistakenly suggested it was).

**Comment by MichaelTrestman** [comment]
Ok, @knz could you supply that release note then please? You are the one who has context on this, correct?

**Comment by knz** [comment]
Release note (security update): the SCRAM authentication and password encryption is not enabled by default.

**Review by MichaelTrestman — COMMENTED** [review]
Thank you.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13781#-:-N1FJDS_CpiPZnoi3G6E:b-9755fa)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox and @knz)

___
*[`_includes/releases/v22.1/v22.1.0-beta.5.md` line 10 at r1](https://reviewable.io/reviews/cockroachdb/docs/13781#-N19MBPlGYlZKlY7TnLy-r1-10:-N1FJJpc7fguC9fIRSiR:b-7rl0jd) ([raw file](https://github.com/cockroachdb/docs/blob/1456202b7e989d23b2efff5c25af03a423a62067/_includes/releases/v22.1/v22.1.0-beta.5.md#L10)):*
<details><summary><i>Previously, knz (kena) wrote…</i></summary><blockquote>

correct no release note needed. You were tagged to ensure that the other PR about scram properly shows that scram is not enabled by default in 22.1 (previous release notes mistakenly suggested it was).
</blockquote></details>

done.

**Review by MichaelTrestman — COMMENTED** [review]
@kathancox I think that's everything. Look good?

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13781#-:-N1FJTnPBxVUcK2Zcygc:berxep4)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox and @knz)

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` [built-in functions](../v22.1/functions-and-operators.html#system-info-functions) now check if user has the [admin role](../v22.1/security-reference/authorization.html#admin-role). [#80384][#80384]
+
+- SCRAM authentication and password encryption are not enabled by default. [#80248][#80248] [7d55af0e6][7d55af0e6] (Raphael 'kena' Poss <knz@cockroachlabs.com>)
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:11]: You can remove this from the end of this note:  "[7d55af0e6][7d55af0e6] (Raphael 'kena' Poss <knz@cockroachlabs.com>)"

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+- [Backups](../v22.1/take-full-and-incremental-backups.html) run by secondary tenants now write protected timestamp records to protect their target schema objects from garbage collection during backup execution. [#80670][#80670]
+
+<h3 id="v22-1-0-beta-5-sql-language-changes">SQL language changes</h3>
+
+- The [cluster settings](../v22.1/cluster-settings.html) `cloudstorage.<provider>.read.node_rate_limit` and `cloudstorage.<provider>.read.node_burst_limit` can now be used to limit throughput when reading from cloud storage during a [`RESTORE`](../v22.1/restore.html) or [`IMPORT`](../v22.1/import.html). [#80281][#80281]
+
+<h3 id="v22-1-0-beta-5-bug-fixes">Bug fixes</h3>
+
+- Fixed bug where automatic encryption-at-rest data key rotation would become disabled after a node restart without a store key rotation. [#80564][#80564]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:23]: Just a typo here, missed "a" from "Fixed a bug"

File: _includes/releases/v22.1/v22.1.0-beta.5.md (hunk 8 above)
Comment (kathancox) [reply _includes/releases/v22.1/v22.1.0-beta.5.md:36]: I think you said there is no note on this one? You can remove this whole line then:

"- [#80641][#80641] [c02b3f015][c02b3f015] release-22.1: update cluster-ui version to v22.1.0 (Marylia Gutierrez <marylia@cockroachlabs.com>)"

File: _includes/releases/v22.1/v22.1.0-beta.5.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` [built-in functions](../v22.1/functions-and-operators.html#system-info-functions) now check if user has the [admin role](../v22.1/security-reference/authorization.html#admin-role). [#80384][#80384]
```
Comment (kathancox) [inline _includes/releases/v22.1/v22.1.0-beta.5.md:9]: Just a small typo here, missed "the" before user:

"... now check if the user has"

**Review by kathancox — APPROVED** [review]
Approved, just need to remove a few of the extra commit and email parts to 2 of the notes and 2 typos. But feel free to merge when you have those fixed up 👍

**Review by MichaelTrestman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13781)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox and @knz)

___
*[`_includes/releases/v22.1/v22.1.0-beta.5.md` line 35 at r1](https://reviewable.io/reviews/cockroachdb/docs/13781#-N19MBPlGYlZKlY7TnLy-r1-35:-N1FO1HyCOBkc_zgHBjz:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/1456202b7e989d23b2efff5c25af03a423a62067/_includes/releases/v22.1/v22.1.0-beta.5.md#L35)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

I think you said there is no note on this one? You can remove this whole line then:

"- [#80641][#80641] [c02b3f015][c02b3f015] release-22.1: update cluster-ui version to v22.1.0 (Marylia Gutierrez <marylia@cockroachlabs.com>)"
</blockquote></details>

Done.
___
*[`_includes/releases/v22.1/v22.1.0-beta.5.md` line 9 at r4](https://reviewable.io/reviews/cockroachdb/docs/13781#-N19MBPlGYlZKlY7TnLy-r4-9:-N1FNmWA8pKeKEmCw-Rp:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/a74e709ba5cccb328a2db062e1df0aac9ee4aaa8/_includes/releases/v22.1/v22.1.0-beta.5.md#L9)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

Just a small typo here, missed "the" before user:

"... now check if the user has"
</blockquote></details>

Done.
___
*[`_includes/releases/v22.1/v22.1.0-beta.5.md` line 11 at r4](https://reviewable.io/reviews/cockroachdb/docs/13781#-N19MBPlGYlZKlY7TnLy-r4-11:-N1FNsD44u8OazpuOarB:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/a74e709ba5cccb328a2db062e1df0aac9ee4aaa8/_includes/releases/v22.1/v22.1.0-beta.5.md#L11)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

You can remove this from the end of this note:  "[7d55af0e6][7d55af0e6] (Raphael 'kena' Poss <knz@cockroachlabs.com>)"
</blockquote></details>

Done.
___
*[`_includes/releases/v22.1/v22.1.0-beta.5.md` line 23 at r4](https://reviewable.io/reviews/cockroachdb/docs/13781#-N19MBPlGYlZKlY7TnLy-r4-23:-N1FNwNS2Q1WwTUuDr8e:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/a74e709ba5cccb328a2db062e1df0aac9ee4aaa8/_includes/releases/v22.1/v22.1.0-beta.5.md#L23)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

Just a typo here, missed "a" from "Fixed a bug"
</blockquote></details>

Done.

**Pull request merged**